ui/imports/utils/Utils.qml

```diff
@@ -811,6 +811,17 @@ QtObject {
             return 34
     }
 
+    function formatCurrencyAmount(currencyAmount, locale) {
+        var amount = parseFloat(currencyAmount.amount)
+        var options = {
+            "minimumFractionDigits": currencyAmount.displayDecimals,
+            "stripTrailingZeroes": currencyAmount.stripTrailingZeroes,
+            "currency": true,
+            "code": currencyAmount.symbol
+        }
+        return Utils.toLocaleString(amount, locale, options)
```

Comment:
argh... should've asked which of the several approaches we currently use for formatting numbers was the actually desired one.

It's easy enough to move some code around and make this function use `LocaleUtils.numberToLocaleString()`.

I think it will prevent future confusion to remove `polyfill.number.toLocaleString.js` and function `Utils.toLocaleString` if we don't want it to be used. I'll create an issue for this if you agree.

Reply:
Thanks! Please burn polyfill.number.toLocaleString.js and `Utils.toLocaleString` with fire :)

Reply:
@caybro please check new function in [ui/imports/utils/LocaleUtils.qml](https://github.com/status-im/status-desktop/pull/8911/files#diff-9350fec6de15101c53e5f1349afddcbf4dd2779c6d04b6430fb79d8efdbd2d8d)
Created new issue to remove the old ones: https://github.com/status-im/status-desktop/issues/8935

Reply:
Thanks!